dts/arm/nxp/nxp_rt7xx_ns.dtsi

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+
+/ {
+	soc {
+		sram: sram@0 {
+			ranges = <0x0 0x0 0x780000
+				0x20000000 0x20000000 0x780000>;
+		};
+
+		peripheral: peripheral@40000000 {
+			ranges = <0x0 0x40000000 0x10000000>;
+		};
+	};
+};
+
+#if defined SOC_MIMXRT798S_CPU0
```

Comment:
DT is processed before kconfig, this is not going to work, and I'm assuming this file has not been actually tested against any build ? if so, just remove this file if not enabling ns in this PR